app/views/dashboards/pro.html.erb

```diff
@@ -4,23 +4,50 @@
   <p>This dashboard will highlight deep insights especially relevant to dev rel.</p>
   <hr />
   <h2>Reactions Summary (❤🦄🔖)</h2>
-  <ul>
-    <li>
-      <%= @this_week_reactions_count %> reactions this week (<%= @last_week_reactions_count.positive? ? (@this_week_reactions_count.to_f / @last_week_reactions_count.to_f) * 100 : "infinity " %>% of previous week)
-    </li>
-    <li>
-      <%= @this_month_reactions_count %> reactions this month (<%= @last_month_reactions_count.positive? ? (@this_month_reactions_count.to_f / @last_month_reactions_count.to_f) * 100 : "infinity " %>% of previous month)
-    </li>
-  </ul>
-    <h2>New Followers Summary (👩‍💻👨‍💻)</h2>
-  <ul>
-    <li>
-      <%= @this_week_followers_count %> new followers this week (<%= @last_week_followers_count.positive? ? (@this_week_followers_count.to_f / @last_week_followers_count.to_f) * 100 : "infinity " %>% of previous week)
-    </li>
-    <li>
-      <%= @this_month_followers_count %> new followers this month (<%= @last_month_followers_count.positive? ? (@this_month_followers_count.to_f / @last_month_followers_count.to_f) * 100 : "infinity " %>% of previous month)
-    </li>
-  </ul>
+    <ul>
+      <li>
+        <%= @this_week_reactions_count %> reactions this week (<%= @last_week_reactions_count.positive? ? (@this_week_reactions_count.to_f / @last_week_reactions_count.to_f) * 100 : "infinity " %>% of previous week)
+      </li>
+      <li>
+        <%= @this_month_reactions_count %> reactions this month (<%= @last_month_reactions_count.positive? ? (@this_month_reactions_count.to_f / @last_month_reactions_count.to_f) * 100 : "infinity " %>% of previous month)
+      </li>
+        <div style="width: 500px; height: 250px; margin: 0 auto;">
+          <canvas
+            id="reactionsChart"
+            data-labels="<%= @this_week_reactions.formatted_dates %>"
+            data-total-count="<%= @this_week_reactions.total_per_day %>"
+            data-total-likes="<%= @this_week_reactions.total_by_type_per_day(type: "Reaction", category: "like") %>"
+            data-total-unicorns="<%= @this_week_reactions.total_by_type_per_day(type: "Reaction", category: "unicorn") %>"
+            data-total-readinglist="<%= @this_week_reactions.total_by_type_per_day(type: "Reaction", category: "readinglist") %>">
+          </canvas>
+        </div>
+      <%= javascript_pack_tag "proCharts", defer: true %>
+    </ul>
+  <h2>New Followers Summary (👩‍💻👨‍💻)</h2>
+    <ul>
+      <li>
+        <%= @this_week_followers_count %> new followers this week (<%= @last_week_followers_count.positive? ? (@this_week_followers_count.to_f / @last_week_followers_count.to_f) * 100 : "infinity " %>% of previous week)
```

Comment:
Wouldn't it be better to round the percentage to 2 decimals? I can add code suggestions for all lines if you agree

```suggestion
        <%= @this_week_followers_count %> new followers this week (<%= @last_week_followers_count.positive? ? ((@this_week_followers_count.to_f / @last_week_followers_count.to_f) * 100).round(2) : "infinity " %>% of previous week)
```

Reply:
Oh yeah, I agree with that. That'd be great :)